napari/_qt/widgets/qt_viewer_dock_widget.py

```diff
@@ -65,7 +73,17 @@ def __init__(
             raise ValueError(f'area argument must be in {list(areas.keys())}')
         self.area = area
         self.qt_area = areas[area]
-        self.shortcut = shortcut
+        if shortcut is not _sentinel:
+            warnings.warn(
```

Comment:
This needs to be localized, also we could just use it one place (if it is the same as below) as class variable.

```python
class QtViewerDockWidget(QDockWidget):

    _SHORTCUT_DEPRECATION_STRING = trans._("...")
```